src/proposal/quorum_threshold.mligo

```diff
@@ -49,7 +49,7 @@ let bound_qt (qt, min_qt, max_qt : quorum_fraction * quorum_fraction * quorum_fr
   else qt
 
 [@inline]
-let period_to_cycle (p: nat): nat = (p + 1n) / 2n
+let stage_to_cycle (p: nat): nat = (p + 1n) / 2n
```

Comment:
Oh somehow I got confused here and didn't update it in my PR.

I think we should add a comment and make it more clear:
```suggestion
// Returns the 'cycle' number of which the given 'stage' number is part of.
let stage_to_cycle (p: nat): nat = (p + 1n) / 2n
```